package.json

```diff
@@ -13,7 +13,7 @@
     "ganache": "./node_modules/.bin/ganache-cli -l 0x663BE0",
     "compile": "rm -rf build/contracts && truffle compile --all",
     "test:unit": "bash scripts/run_ganache.sh && truffle test test/unit/*",
-    "test:integration": "truffle test test/integration/*",
+    "test:integration": "truffle test test/integration/Delegation.js && truffle test test/integration/JobAssignment.js",
```

Comment:
This is a hack to get the integration tests to pass - Truffle is supposed to redeploy all contracts between test runs to guarantee a clean room environment, but this does not seem to be working for some reason so for the time being this NPM script will manually invoke each integration test

Reply:
Never mind this doesn't play well with the CI - will use a different fix

Reply:
Switched the integration tests to use ganache-cli for now because there seems to be a bug in Truffle with redeploying contracts between test runs which I've filed an issue for